src/test/java/com/exasol/adapter/dialects/IntegrationTestConstants.java

```diff
@@ -1,9 +1,29 @@
 package com.exasol.adapter.dialects;
 
+import java.nio.file.Path;
+
 public final class IntegrationTestConstants {
     public static final String INTEGRATION_TEST_CONFIGURATION_FILE_PROPERTY = "integrationtest.configfile";
+    // Exasol
```

Comment:
You should separate the constants for different products. This breaks the SRP.

Reply:
moved them